spring-expression/src/main/java/org/springframework/expression/spel/ast/SpelNodeImpl.java

```diff
@@ -214,62 +213,67 @@ protected ValueRef getValueRef(ExpressionState state) throws EvaluationException
 	 * and if it is then the argument values will be appropriately packaged into an array.
 	 * @param mv the method visitor where code should be generated
 	 * @param cf the current codeflow
-	 * @param member the method or constructor for which arguments are being set up
+	 * @param executable the method or constructor for which arguments are being set up
 	 * @param arguments the expression nodes for the expression supplied argument values
 	 */
-	protected static void generateCodeForArguments(MethodVisitor mv, CodeFlow cf, Member member, SpelNodeImpl[] arguments) {
```

Comment:
Since this is a `protected` method in a `public` type, we cannot simply change the method signature.

Instead, we would need to introduce an overloaded variant that accepts `Executable` and deprecate the existing variant that accepts `Member`.